### PR TITLE
issue#4

### PR DESCRIPTION
**Pull request by robertb98**
login - /api/auth/jwt/create/
refresh - /api/auth/jwt/refresh
me - /api/auth/users/me/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -141,4 +141,4 @@ dmypy.json
 # End of https://www.toptal.com/developers/gitignore/api/python
 /.vscode
 /.pytest_cache
-Pipfile.lock
```
Comment (nitz14) [inline .gitignore:144]: missing empty line

File: backend/Dockerfile [hunk 2]
```diff
@@ -11,5 +11,5 @@ RUN pip install pipenv && pipenv install
 
 COPY . /backend
 
-EXPOSE 8000
+# EXPOSE 8000
```
Comment (nitz14) [inline backend/Dockerfile:14]: remove this line

**Comment by robertmusil** [comment]
Seems like this PR has a lot of changes covered by #7

**Comment by robertb98** [comment]
> Seems like this PR has a lot of changes covered by #7

Yup, something went wrong when I wanted to rebase changes from previous branch. Will fix it.

**Review by nitz14 — APPROVED** [review]

**Comment by robertmusil** [comment]
Please rebase

**Review by robertmusil — APPROVED** [review]

**Pull request closed without merging**